### PR TITLE
Fix multi-valued attribute return type issue

### PR DESCRIPTION
**Pull request by gayashanbc**
Resolves https://github.com/wso2/product-is/issues/5209

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ target
 *.iws
 *.ipr
 .idea
+*DS_Store
```
Comment (inthirakumaaran) [inline .gitignore]: Add a dot in front of a file extension

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java [hunk 2]
```diff
@@ -110,11 +112,14 @@ else if (attribute instanceof ComplexAttribute) {
     }
 
     /**
-     * set claim mapping for simple attribute
+     * set claim mapping for simple attribute.
```
Comment (inthirakumaaran) [inline components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java]: Check the capitalization for the comments in all the places.

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java [hunk 3]
```diff
@@ -704,18 +758,20 @@ public static void constructSCIMObjectFromAttributesOfLevelThree(Map.Entry<Strin
     }
 
     /**
-     * return the attribute schema for the asked attribute URI
+     * return the attribute schema for the asked attribute URI.
+     *
      * @param attributeURI
      * @param scimObjectType
      * @return
```
Comment (inthirakumaaran) [inline components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java]: Add parameter comments.

File: .gitignore (hunk 1 above)
Comment (gayashanbc) [reply .gitignore]: Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/d14dfd5420cd70c2bb864885bcb9ab336dca6ac0

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java (hunk 2 above)
Comment (gayashanbc) [reply components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java]: Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/d14dfd5420cd70c2bb864885bcb9ab336dca6ac0

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java (hunk 3 above)
Comment (gayashanbc) [reply components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java]: Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/d14dfd5420cd70c2bb864885bcb9ab336dca6ac0

**Pull request merged**